bin/calyptos

```diff
@@ -3,8 +3,28 @@ import argparse
 from fabric.colors import yellow
 from stevedore import driver, extension
 from calyptos.rolebuilder import RoleBuilder
+import os
+import shutil
+
+dot_dir = os.path.expanduser('~/.calyptos')
+default_cfgfile = os.path.join(dot_dir + "/" + "config.yml")
+
+def locate_cfgfile():
+    if not os.path.isfile(default_cfgfile):
```

Comment:
I would create an array of paths to check, then iterate over them in order until I found one and use that as the default.